QuizApp/Data/Quiz/QuizRepository.swift

```diff
@@ -0,0 +1,23 @@
+class QuizRepository: QuizRepositoryProtocol {
+
+    private let quizNetworkClient: QuizNetworkClientProtocol
+
+    init(quizNetworkClient: QuizNetworkClientProtocol) {
+        self.quizNetworkClient = quizNetworkClient
+    }
+
+    var quizes: [QuizRepoModel] {
+        get async throws {
+            let quizesResponse = try await quizNetworkClient.quizes
```

Comment:
You can shorten this to `try await quizNetworkClient.quizes.map { QuizRepoModel(from: $0) }`